easybuild/easyconfigs/g/GAMESS-US/GAMESS-US-20130501-R1-intel-2015a.eb

```diff
@@ -19,5 +19,7 @@ patches = ['GAMESS-US_rungms-slurm.patch']
 # it's OK if these values are larger than what your system provides
 maxcpus = '1000'
 maxnodes = '100000'
+scratch_dir = '$SCRATCH_DIR'
```

Comment:
Can the value of scratch_dir be overridden? I'm OK with just having a "local" easyconfig if that would be best.